src/player.cpp

```diff
@@ -11325,6 +11358,31 @@ bool player::sees( const Creature &critter ) const
     return Creature::sees( critter );
 }
 
+float player::get_sees_modifier() const
+{
+    float modifier = 1.0;
+    if ( get_size() == MS_SMALL ) {
+        // player sillhouete is 50% smaller while sneaking
+        modifier = 0.5;
+    }
+
+    // having moved during last action adds 12.5% to 50% depending on movecost (slower is better)
+    if ( moved ) {
+        modifier += ( 0.25 / ( movecounter / 100.0 ) );
+    }
+
+    // survival skill helps with staying hidden in all circumstances (2.5% per level)
+    modifier -= get_skill_level( skill_survival ) * 0.025;
+
+    // encumbrance on bodyparts has negative effect (every 20 points = 1%)
+    modifier += (float) ( encumb( bp_torso ) + encumb( bp_head ) +
```

Comment:
That cast is unnecessary, the right side of the division is already a floating point value (`2000.0`), the left side will be converted to a floating point value automatically.

Reply:
Will change that, thank you